### DISCUSSION TITLE
distill -  code block not displaying as intended

### DISCUSSION THREAD

**Discussion by hrhoo** [alshedivat/al-folio#2632] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e620ec0b-e779-49cc-9eaa-dc8093a1067f)

![image](https://github.com/user-attachments/assets/401ec6b2-0591-4f3f-a124-5df35cc1dbe7)

This is from _config.yml
Is there anything I need to do?
I don't know why it works this way..

I saw a post from issues. I guess code highlight is not working on distill??

**Comment by george-gca** [comment, marked as answer]
Check https://github.com/alshedivat/al-folio/issues/1220#issuecomment-2211979814 to see if it helps.

**Reply by hrhoo** [reply]
Do you think there would be some update for distill.
it seems like assets/js/copy_code.js is not working for distill as well..

I don't know if this is a bug or it is intended.

**Reply by george-gca** [reply]
Maybe this is an easy fix by just importing the `copy_code.js` in distill template as it is done in the default layout. If this fixes it, would you mind sending a PR for this?

**Reply by hrhoo** [reply]
![image](https://github.com/user-attachments/assets/3a6820d0-9584-477d-980a-2a239040b7df)

I am not sure how distill works and is linked with other elements...
I assume I am supposed to import `copy_code.js` in `\_layouts\distill.liquid`.
It seems like it doesn't work.

**Reply by hrhoo** [reply]
prism doesn't seem to work well on distill (I don't know how to fix...)
However, It seems like the easy way to stop prism highlighting code blocks is to edit this line a bit..
so prism can't find code blocks.

This can be found in `\assets\js\distillpub\template.v2.js`
![image](https://github.com/user-attachments/assets/6e65d73a-0432-405e-9efa-abecb7c079cf)

Still copy_code.js is not working in distill
